src/main/java/weiweibot/commands/DeleteCommand.java

```diff
@@ -0,0 +1,36 @@
+package weiweibot.commands;
+
+import weiweibot.storage.Storage;
+import weiweibot.tasks.Task;
+import weiweibot.tasks.TaskList;
+
+
+/**
+ * Deletes the task at a given zero-based index and saves the updated list.
+ *
+ * <p>Side effects: removes an item from {@link TaskList}, calls
+ * {@link Storage#save(TaskList)}, and prints a short confirmation message.</p>
+ *
+ * <p>Errors from {@link TaskList#deleteTask(int)} (e.g., out-of-bounds) are
+ * propagated to the caller.</p>
+ */
+public class DeleteCommand extends Command {
+    private final int indexZeroBased;
```

Comment:
It would be better practice for constants like these to be named in SCREAMING_SNAKE_CASE